package.json

```diff
@@ -1,28 +1,27 @@
 {
 	"private": true,
 	"scripts": {
-		"bootstrap": "lerna bootstrap",
-		"build": "lerna run build",
-		"clean": "lerna run clean",
-		"dev": "lerna run --parallel dev",
+		"build": "tsc -b",
+		"dev": "yarn build -w",
+		"clean": "rimraf 'packages/**/{lib,*.tsbuildinfo}'",
 		"test": "jest --coverage",
 		"test:coverage": "jest --coverage --coverageReporters=text-lcov | coveralls",
 		"test:clear": "jest --clearCache",
 		"test:unit": "jest",
 		"test:heapdump": "node --expose-gc ./heaptest/index.js",
-		"lint": "prettier --write './{packages,website,playground}/**/*.{ts,tsx,json}'; eslint --fix './{packages,website,playground}/**/*.{ts,tsx}'; yarn cli './website/**/*.tsx'",
-		"schema": "lerna run schema",
-		"gen": "cd packages/\\@markuplint/html-spec/; yarn gen; cd ../",
-		"all": "yarn lint; yarn clean; yarn gen; yarn build; yarn schema; yarn test:clear; yarn test;",
+		"lint": "prettier --write .; eslint . --fix; yarn cli './website/**/*.tsx'",
```

Comment:
All source codes except test fixtures should be formatted and linted.

`.prettierignore` and `.eslintignore` makes sure IDE works correctly too.